.github/workflows/scala.yml

```diff
@@ -7,10 +7,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Set up JDK 1.8
-        uses: olafurpg/setup-scala@v2
+        uses: olafurpg/setup-scala@v7
```

Comment:
We need something like scala-steward to tell us when these get updated, I guess.

Reply:
Right, I discovered the update just by chance